Presentation.Web/Controllers/API/V1/AuthorizeController.cs

```diff
@@ -62,10 +66,21 @@ public HttpResponseMessage GetLogin()
 
         [Route("api/authorize/GetOrganizations")]
         [SwaggerResponse(HttpStatusCode.OK, Type = typeof(ApiReturnDTO<IEnumerable<OrganizationSimpleDTO>>))]
-        public HttpResponseMessage GetOrganizations()
+        public HttpResponseMessage GetOrganizations([FromUri]string orderBy = null, [FromUri]bool? orderByAsc = true)
         {
             var orgs = GetOrganizationsWithMembershipAccess();
 
+            if (!string.IsNullOrEmpty(orderBy))
+            {
+                if (!string.Equals(orderBy, "Name"))
+                    return BadRequest("Incorrect OrderBy Property name");
+
+                orderByAsc ??= true;
+                
+                orgs = orderByAsc.Value ? orgs.OrderBy(orderBy)
```

Comment:
I woul rather that you either build up an expression (using the expression type that it expects in stead of passing strings and then compiling an expression)

it would be

`Expression<Func<Organization,object>> = org => org.Name`

where the important part is here: `Expression<Func<Organization,object>>`
you can then pass that to the orderByAsc | orderbyDesc methods

Reply:
Sure, I just thought that the method parameter "orderBy" should contain the name of the property which later is used to sort the list (for now it was meant to just be "Name"), that's why I created the Extension class, so that it gets the property by name, and then it uses the property to sort the list